Dockerfile

```diff
@@ -16,9 +15,13 @@ RUN mkdir -p ${TMP_DIR} && \
     # already create volume dir to later have correct rights
     mkdir ${TMP_DIR}/data
 
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
+
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ${TMP_DIR}/bin/tini
 COPY docker/utils/startup.sh ${TMP_DIR}/bin/startup.sh
 RUN chmod +x -R ${TMP_DIR}/bin/
+RUN chown 1000:1000 -R ${TMP_DIR}/ && chmod -R 0775 ${TMP_DIR}/ ${TMP_DIR}/data
```

Comment:
I don't think we need to set the owner in the builder image as we set them on copy to the final image